src/main/content/_assets/js/instance-common.js

```diff
@@ -178,11 +216,15 @@ function updateInstanceAdminView(adminMembersJson) {
                     $(userInfoBox).find(".bx--assistive-text").text(`Remove ${userLogin} from ${team.name} team`)
                 })
         });
+
         $("#admin-modal-list").append(row);
 
+        let addUserInputText = $("#addUser-input-template").clone().removeAttr("id").removeClass("hidden");
+        $(addUserInputText).find(".user-add-team-name").text(team.name)
+        addUserInputText.insertAfter($(row).find(".bx--inline-notification").last())
     });
 
-    let uniqueAdminList = Object.values(instanceAdmins.reduce((accumulator, curretValue) => Object.assign(accumulator, { [curretValue.login]: curretValue}), {}));
+    let uniqueAdminList = Object.values(instanceAdmins.reduce((accumulator, curretValue) => Object.assign(accumulator, { [curretValue.login]: curretValue }), {}));
```

Comment:
you might not need Object.values, Object.assign, and reduce all in the same line. Maybe an easier way? what does instanceAdmins look like? I assume you are just removing duplicates?

Maybe take a look at [Set](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Set)